spec/webview-spec.js

```diff
@@ -285,15 +285,17 @@ describe('<webview> tag', function () {
   describe('httpreferrer attribute', () => {
     it('sets the referrer url', (done) => {
       const referrer = 'http://github.com/'
-      const listener = (e) => {
-        assert.equal(e.message, referrer)
-        webview.removeEventListener('console-message', listener)
+      const server = http.createServer((req, res) => {
+        assert.equal(req.headers.referer, referrer)
```

Comment:
If this fails the `assert.equal` will throw and the server won't be stopped.

Reply:
Would using a `try { } finally { }` be preferred here ?

Reply:
Can you stop the server before calling `assert.equals`? )

Reply:
Yup can do that, its better solution here.